mappings/net/minecraft/util/math/layer/InitLayer.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS cfo net/minecraft/util/math/layer/InitLayer
```

Comment:
These should be in `net/minecraft/world/biome/layer/`.

Reply:
I did consider this, though `InitLayer` is a base interface for any layer that does not have a parent, so it isn't really tied to the biome system.

Reply:
The entire system is for biome layers, there's zero point having this separate.